package.json

```diff
@@ -24,9 +24,7 @@
     "src/**/*.js",
     "src/**/*.d.ts",
     "!src/**/*.d.js",
-    "!src/**/web",
     "!src/**/web_hammer",
```

Comment:
Unsure what `web_hammer` is and whether treasury wallet needs it.

Reply:
looks like some stuff on top of a fork of https://github.com/hammerjs/hammer.js for touch gestures

Reply:
so it works fine without this dir?

Reply:
It does not work without it. Removed that exclusion line as well.